spec/models/medical_history_spec.rb

```diff
@@ -47,4 +47,34 @@
       expect(patient.medical_history.indicates_risk?).to eq(true)
     end
   end
+
+  describe '#indicates_hypertension_risk?' do
+    it 'returns true if there was a prior heart attack' do
+      patient = create(:patient)
+      create(:medical_history, prior_heart_attack_boolean: true, patient: patient)
+
+      expect(patient.medical_history.indicates_hypertension_risk?).to eq(true)
+    end
+
+    it 'returns true if there was a prior stroke' do
+      patient = create(:patient)
+      create(:medical_history, prior_stroke_boolean: true, patient: patient)
+
+      expect(patient.medical_history.indicates_hypertension_risk?).to eq(true)
+    end
+
+    it 'returns false if there is diabetes history' do
```

Comment:
Since the method can return `nil`, which I presume why we've chosen `falsey` in the assertion, we can be clear about that in the `it` description as well.

Reply:
Whoops, forgot to update the spec description.

Reply:
Whoops, forgot to update the spec description.